drivers/can/can_stm32fd.c

```diff
@@ -56,12 +60,19 @@ static int can_stm32fd_get_core_clock(const struct device *dev, uint32_t *rate)
 
 static void can_stm32fd_clock_enable(void)
 {
-#ifdef CONFIG_SOC_SERIES_STM32U5X
+	LL_RCC_SetFDCANClockSource(CAN_STM32FD_CLOCK_SOURCE);
+
+	/* LL_RCC API names do not align with PLL output name but are correct */
+	/* Should these be enabled here or elsewhere by a clock tree? */
```

Comment:
You can remove this line. Transition to device is in the pipe.

Reply:
The comment line, correct?

Reply:
Right